index.php

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <?php $base = "../../" ?>
+    <base href="../../">
+    <script src="js/jquery-2.2.4.min.js"></script>
+    <script src="js/facebox.js"></script>
+    <script src="js/gameSettings.js"></script>
+    <link rel="stylesheet" type="text/css" href="css/facebox.css"/>
+    <link rel="stylesheet" type="text/css" href="css/main.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.css"/>
```

Comment:
or any of these stylesheets? is this for the initial page?

Reply:
Yes, it's used for the initial page to have a similar style as the other Dr. Ecco pages.